common/src/main/java/org/apache/atlas/service/redis/RedisServiceImpl.java

```diff
@@ -2,24 +2,90 @@
 
 import org.apache.atlas.AtlasException;
 import org.apache.atlas.annotation.ConditionalOnAtlasProperty;
+import org.apache.atlas.service.metrics.MetricUtils;
 import org.redisson.Redisson;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
 import org.springframework.stereotype.Component;
 
 import javax.annotation.PostConstruct;
 
 @Component
 @ConditionalOnAtlasProperty(property = "atlas.redis.service.impl")
+@Order(Ordered.HIGHEST_PRECEDENCE)
```

Comment:
Thisb never worked for me, did you check whether this is initialized before typeDefs are bootstrapped?

Reply:
It is fine if It it is loaded before AtlasTypeDefGraphStoreV2